### ISSUE TITLE
Multiple validation datasets

### ISSUE THREAD

**Issue by RaymondLi0** [bigcode-project/Megatron-LM#29]
Add a feature to run validation on multiple datasets.
Would be useful to get validation loss on different programming languages for example

**Comment by RaymondLi0** [comment]
This could be in the form of adding just one argument, specifying that datasets should not be blended for evaluation?

**Comment by RaymondLi0** [comment]
This is implemented in bigscience's Megatron-Deepspeed:

- Build the list of valid dataloaders: https://github.com/bigscience-workshop/Megatron-DeepSpeed/blob/e52bdabbde3c6895aceb76c1bced295c2646121f/megatron/training.py#L1209
- evaluate over all valid datasets: https://github.com/bigscience-workshop/Megatron-DeepSpeed/blob/e52bdabbde3c6895aceb76c1bced295c2646121f/megatron/training.py#L972

**Comment by RaymondLi0** [comment]
implemented in #32
